packages/nextjs/app/large-grant-apply/page.tsx

```diff
@@ -138,23 +138,15 @@ const LargeGrantApply: NextPage = () => {
                     <FormTextarea
                       label="Description"
                       showMessageLength
-                      {...formMethods.register(`milestones.${index}.description`)}
-                      error={errors?.milestones?.[index]?.description?.message}
+                      {...getCommonOptions(`milestones.${index}.description`)}
```

Comment:
It's not showing the error message on milestone **Description** and **Detail of Deliverables** when you leave the field blank or with less than 20 chars.